### PR TITLE
Update DOCS.md

### PR DESCRIPTION
**Pull request by briodan**
added copy and paste instructions for the Web UI.

### CODE REVIEW

File: ssh/DOCS.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ ha help
 
 You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.
 
+To copy text from the Web UI:
+- hold down the SHIFT key
+- select the text you want to copy
+- when you release the left mouse button the text will copy to your system clipboard
+
+To paste text in the Web UI press SHIFT + INSERT
```
Comment (rytilahti) [inline ssh/DOCS.md:37]:
```suggestion
You can copy text by holding down the shift key and then selecting the text to copy. Releasing the mouse button will copy the text to your system clipboard.
Use SHIFT + INSERT to paste text to the terminal.
```
Rewording suggestion.

File: ssh/DOCS.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ ha help
 
 You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.
 
+To copy text from the Web UI:
+- hold down the SHIFT key
+- select the text you want to copy
+- when you release the left mouse button the text will copy to your system clipboard
+
```
Comment (agners) [inline ssh/DOCS.md:36]:
```suggestion
- select the text you want to copy using your mouse
- releasing the left mouse button will copy the text to your system clipboard
```

File: ssh/DOCS.md (hunk 1 above)
Comment (agners) [reply ssh/DOCS.md:37]: I think the step by step style is kinda nice. But I like some your suggestion style wise. Made a separate suggestion.

File: ssh/DOCS.md [hunk 3]
```diff
@@ -29,6 +29,13 @@ ha help
 
 You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.
 
+To copy text from the Web UI:
+- hold down the SHIFT key
+- select the text you want to copy
+- when you release the left mouse button the text will copy to your system clipboard
```
Comment (agners) [inline ssh/DOCS.md:35]:
```suggestion
- select the text you want to copy using your mouse
- releasing the left mouse button will copy the text to your system clipboard
```

File: ssh/DOCS.md [hunk 4]
```diff
@@ -29,6 +29,13 @@ ha help
 
 You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.
 
+To copy text from the Web UI:
+- hold down the SHIFT key
+- select the text you want to copy using your mouse
+- releasing the left mouse button will copy the text to your system clipboard
+
+To paste text in the Web UI press SHIFT + INSERT
```
Comment (c0ffeeca7) [inline ssh/DOCS.md:37]:
```suggestion
1. Hold down the SHIFT key.
2. Select the text you want to copy using your mouse.
3. To copy the text to your system clipboard, release the left mouse button.
4. To paste text in the Web UI, press SHIFT + INSERT.
```

**Review by c0ffeeca7 — COMMENTED** [review]
convert to numbered list, as this procedure is sequential

File: ssh/DOCS.md (hunk 4 above)
Comment (agners) [reply ssh/DOCS.md:37]: I think copy and pasting text are (potentially) separate things you want to do in the shell, so I would not lump those together in one sequence.

Comment (c0ffeeca7) [reply ssh/DOCS.md]: right

Comment (agners) [reply ssh/DOCS.md]: One could argue that "select the text" already contains the left mouse release... But this is maybe a bit detail. I kinda liked the passive mood though for that reason, just explaining when the copying exactly happens. But it gets into details here :sweat_smile:

**Review by c0ffeeca7 — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks all! 👍 

../Frenck

**Pull request merged**